src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandResult.java

```diff
@@ -173,6 +173,7 @@ public record Error(
       String message,
       @JsonAnyGetter @Schema(hidden = true) Map<String, Object> fields,
       // Http status to be used in the response, defaulted to 200
```

Comment:
Need to move down by one line, to be right before `status` field

Reply:
fixed, thanks!